T012_M3_text_ui

```diff
@@ -0,0 +1,189 @@
+            # Sort through and find which attribute the user wanted to load data by, and then print a statement letting the user know the data has been loaded
+            if cmd2 == "Failures":
+                loaded_data = T012_M1_load_data.add_average(
+                    T012_M1_load_data.student_failures_dictionary(filename))
+                print("Data loaded" + "\n")
+            elif cmd2 == "Health":
+                loaded_data = T012_M1_load_data.add_average(
+                    T012_M1_load_data.student_health_dictionary(filename))
+                print("Data loaded" + "\n")
+            elif cmd2 == "School":
+                loaded_data = T012_M1_load_data.add_average(
+                    T012_M1_load_data.student_school_dictionary(filename))
+                print("Data loaded" + "\n")
+            elif cmd2 == "Age":
+                loaded_data = T012_M1_load_data.add_average(
+                    T012_M1_load_data.student_age_dictionary(filename))
+                print("Data loaded" + "\n")
+            else:
+                # If an error occurs, print that an error occured
+                print("\n" + "Load data error" + "\n")
+
+        elif (cmd == "S"):  # Sort Data command
+            # Create a variable of the sorted data to be accessed throughout this if statement
+            sorted_data = []
+            if loaded_data == {}:  # Error preventative if statement incase loaded data hasn't been loaded yet
+                print("File not loaded. Please, load a file first.")
+            else:
+                cmd3 = input(
+                    "Please enter the attribute you want to use for sorting (Options: School, Age, StudyTime, Failures, Health, Absences, G1, G2, G3, G_Avg): ")  # Get sorting attribute from the user
+                # If the attribute the user entered is invalid, prompt user to try again
+                while (cmd3 not in supported_cmds):
+                    cmd3 = input(
+                        "Invalid command. Please enter the attribute to use as key for sorting: ")
+                sorted_data = (T012_M2_sort_plot.sort_students_selection(
+                    loaded_data, cmd3))  # Store the sorted data
+                cmd4 = (input("sorted_data loaded, and sorted by " + str(cmd3) +
+                              ". Do you want to display the data? (Options: Yes or No): ")).capitalize()  # Ask the user if they want to display the data
+                # If user enters invalid command, prompt to retry
+                while (cmd4 not in supported_cmds):
+                    cmd4 = (input(
+                        "Invalid command. Do you want to display the data? (Options: Yes or No): ")).capitalize()
+                if cmd4 == "Yes":
+                    # If the user wants to display the data, print the sorted data.
+                    print(str(sorted_data) + "\n")
```

Comment:
```suggestion
                    for i in iter(sorted_data):
                        print(i, ":", sep="")
                        for j in range(len(sorted_data[i])):
                            print("\t", sorted_data[i][j], sep="")
```

Reply:
I didn't read the descriptions for the functions you and Esteban had to write .. so idk if you're allowed to this or you had to do it the way you had it.. throw it in your code and see if you like?